MakairaConnect.php

```diff
@@ -47,9 +48,29 @@ public function build(ContainerBuilder $container)
      */
     public function install(InstallContext $installContext)
     {
+        $this->checkSchemaStructure();
         $this->installModels();
     }
 
+    public function update(UpdateContext $context)
+    {
+
+        $this->checkSchemaStructure();
+        parent::update($context);
+    }
+
+    private function checkSchemaStructure()
+    {
+        $conn = $this->container->get('dbal_connection');
+        $state = $conn->query(
+            "SELECT SCHEMA_NAME FROM information_schema.SCHEMATA WHERE SCHEMA_NAME = 'mak_revision' LIMIT 1"
+        );
+
+        if ($state->fetchColumn()) {
+            $conn->exec('ALTER TABLE mak_revision RENAME TO s_plugin_makaira_connect_revision');
```

Comment:
Maybe consider creating a constant for the revision table name because you are using it in many places.

Reply:
In my opinion there is no need for a constant. The name won't change and the table name is used at two places.